doc/source/user_guide/groupby.rst

```diff
@@ -568,6 +568,67 @@ For a grouped ``DataFrame``, you can rename in a similar manner:
                             'mean': 'bar',
                             'std': 'baz'}))
 
+.. _groupby.aggregate.named:
+
+Named Aggregation
+~~~~~~~~~~~~~~~~~
+
+.. versionadded:: 0.25.0
+
+To support column-specific aggregation *with control over the output column names*, pandas
+accepts the special syntax in :meth:`GroupBy.agg`, known as "named aggregation", where
+
+- The keywords are the *output* column names
+- The values are tuples whose first element is the column to select
+  and the second element is the aggregation to apply to that column. Pandas
+  provides the ``pandas.NamedAgg`` namedtuple with the fields ``['column', 'aggfunc']``
```

Comment:
can give a ``:class:`NamedAgg` `` ?

Reply:
is this added in the reference.rst ?

Reply:
The docstring autogenerated by the namedtuple isn't super helpful

![Screen Shot 2019-05-28 at 9 24 37 AM](https://user-images.githubusercontent.com/1312546/58486358-54ee7000-812b-11e9-9ad2-092a53bf7b42.png)